chat.go

```diff
@@ -173,6 +244,33 @@ func (api *Client) SendMessageContext(ctx context.Context, channelID string, opt
 	return response.Channel, response.getMessageTimestamp(), response.Text, response.Err()
 }
 
+// DeleteScheduledMessage removes a scheduled message
+// Note that messages scheduled to be sent within the next 60 seconds will fail to delete as they
+// are already being processed by Slack.  You will receive a failure message stating the ID does not
+// exist.
+func (api *Client) DeleteScheduledMessage(channelID, scheduledMessageID string) (string, string, error) {
+	respChannel, respTimestamp, err := api.DeleteScheduledMessageContext(
+		context.Background(),
+		channelID,
+		scheduledMessageID,
+	)
+	return respChannel, respTimestamp, err
+
+}
+
+// DeleteScheduledMessageContext removes a scheduled message with custom context.
+// See DeleteScheduledMessage for more information
+func (api *Client) DeleteScheduledMessageContext(ctx context.Context, channelID, scheduledMessageID string) (string, string, error) {
+	respChannel, respTimestamp, _, err := api.SendMessageContext(
+		ctx,
+		channelID,
+		MsgOptionDeleteSchedule(scheduledMessageID),
+		MsgOptionCompose(),
```

Comment:
eh? this option doesn't do anything?

Reply:
Probably a bad copy and paste, really can't explain it.  Fixed.